dist/images/env-check.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set +e
+checkresult=/var/log/kube-ovn/env-check.log
+echo "Start environment check" > $checkresult
+echo "1) check cni configuration" >> $checkresult
```

Comment:
Considering the manual execution scenarios, we should log the result by `bash env-check.sh | tee /var/log/kube-ovn/env-check.log`.

Reply:
modified